ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -168,6 +169,7 @@ private PeerDiscoveryController(
     this.outboundMessageHandler = outboundMessageHandler;
     this.peerBondedObservers = peerBondedObservers;
     this.discoveryProtocolLogger = new DiscoveryProtocolLogger(metricsSystem);
+    this.bondingPeers = new HashMap<>();
```

Comment:
is it not necessary to use a ConcurrentHashMap here?

Reply:
I don't think we need that here. My understanding is that each peer has it's own controller so the Map should be ok.